config/locales/candidate_interface/review_application.yml

```
@@ -69,6 +69,13 @@ en:
     references:
       incomplete: Add %{minimum_references} referees to your application
       complete_section: Add your references
+    new_references:
+      incomplete: References section not marked as complete
+      complete_section: Complete references section
+      one_reference_only: You need to add a second reference
+      add_more_references: Add references
```

I think we should say "Add another reference" here, for consistency with the button text in the section itself.

```suggestion
      add_more_references: Add another reference
```

---

👍